src/main/kotlin/org/rust/cargo/runconfig/CargoRunConfigurationBase.kt

```diff
@@ -46,14 +56,11 @@ class CargoCommandConfiguration(project: Project,
         }
     }
 
-    override fun getConfigurationEditor(): SettingsEditor<out RunConfiguration> =
-        CargoRunConfigurationEditorForm()
-
     override fun getState(executor: Executor, environment: ExecutionEnvironment): RunProfileState? {
         val module = configurationModule.module ?: return null
         val toolchain = module.toolchain ?: return null
         val moduleDirectory = module.cargoProjectRoot ?: return null
-        val args = ParametersListUtil.parse(additionalArguments)
+        val args = ParametersListUtil.parse(arguments + additionalArguments)
```

Comment:
I think this `arguments + additionalArguments` is not quite correct. If `arguments` is `foo`, and `additionalArguments` is `bar`, then their concatenation is `foobar` and not `foo bar`. 

I think that the simplest thing to do now is to remove the `arguments` field and `CargoTestRunConfiguration` altogether. It would be easier to add them after we merge the rest of this PR. 

The end design could look like this:

```Kotlin
// in base run configuration
// This is the interface for inheritors, and we don't show this field in
// GUI directly
protected open val arguments: List<String>

// in derived configuration, i.e, Test
override val arguments = listOf("--test", testName)

// This is what we show in GUI. 
var testName: String = ""

```